client/src/products/Product.tsx

```diff
@@ -171,62 +186,88 @@ const Product = () => {
+                    />
+                    <Button
+                      className="fit-content"
+                      variant="tertiary"
+                      icon={<FloppydiskIcon fontSize="1.5rem" aria-hidden />}
+                      onClick={handleSaveProductTitle}
                     >
-                      {series.title ?? ""}
-                    </a>
-                  ) : (
-                    <>{series.title ?? ""}</>
-                  )}
-                </Heading>
-              )}
+                      Lagre
+                    </Button>
+                  </>
+                )}
 
-              {showEditProductTitleMode && (
-                <>
-                  <TextField
-                    defaultValue={series.title ?? ""}
-                    label={""}
-                    aria-label="Rediger tittel"
-                    id="title"
-                    name="title"
-                    onChange={(event) => setProductTitle(event.currentTarget.value)}
-                  />
+                {isEditable && !showEditProductTitleMode && (
                   <Button
                     className="fit-content"
                     variant="tertiary"
-                    icon={<FloppydiskIcon fontSize="1.5rem" aria-hidden />}
-                    onClick={handleSaveProductTitle}
-                  >
-                    Lagre
-                  </Button>
-                </>
-              )}
-
-              {isEditable && !showEditProductTitleMode && (
-                <Button
-                  className="fit-content"
-                  variant="tertiary"
-                  icon={<PencilWritingIcon title="Endre produktnavn" fontSize="1.5rem" />}
-                  onClick={() => {
-                    setProductTitle(series.title);
-                    setShowEditProductTitleMode(true);
-                  }}
-                ></Button>
+                    icon={<PencilWritingIcon title="Endre produktnavn" fontSize="1.5rem" />}
+                    onClick={() => {
+                      setProductTitle(series.title);
+                      setShowEditProductTitleMode(true);
+                    }}
+                  ></Button>
+                )}
+              </HStack>
+              {seriesStatus(series) !== SeriesStatus.PUBLISHED && (
+                <Box
+                  paddingInline="4"
+                  paddingBlock="1"
+                  borderRadius="small"
+                  width="fit-content"
+                  style={{ backgroundColor: "#EFECF4" }}
+                >
+                  <HStack gap="2" align="center">
+                    <svg width="6" height="6" viewBox="0 0 6 6" fill="none">
+                      <circle cx="3" cy="3.5" r="3" fill="#8269A2" />
+                    </svg>
+                    <BodyShort>{isPublished ? "Nye endringer" : "Nytt produkt"}</BodyShort>
+                  </HStack>
+                </Box>
```

Comment:
Dette er nytt

Reply:
Syns dette var bra info nå. I framtida kan vi vurdere å sjekke diff også for å sjå om det faktisk er endringar eller om produktet bare er satt i endringsmodus (men i 99,9% av tilfella gjer vel folk faktiske endringar ganske kjapt slik at tagge vil stemme med virkeligheten)